src/Cache.php

```diff
@@ -215,9 +270,34 @@ protected function setValues($data, $expire)
             }
         }
 
+        $this->hashTagAvailable = false;
+
         return $failedKeys;
     }
 
+    public function getIsCluster()
+    {
+        if ($this->forceClusterMode !== null) {
+            return $this->forceClusterMode;
+        }
+
+        if ($this->_isCluster === null) {
+            $this->_isCluster = false;
+            try {
+                /**
+                 * if redis is running without cluster support, this command results in:
+                 * `ERR This instance has cluster support disabled`
+                 * and [[Connection::executeCommand]] throws an exception
+                 */
+                $this->redis->executeCommand('CLUSTER INFO');
+                $this->_isCluster = true;
+            } catch (Exception $exception) {
```

Comment:
If the intent is to ignore every exception possible then it should be added as comment inside `catch`

Reply:
The intention is to catch this exception: https://github.com/yiisoft/yii2-redis/blob/a456513924fa448ac84bdb2d6c200f194509bce2/src/Connection.php#L746

I didn't want not change the way `parseResponse` handles actual errors. The comment in the try explains what I'm doing.